units/host_monitoring_idle_termination.go

```diff
@@ -25,7 +25,9 @@ const (
 	idleHostJobName = "idle-host-termination"
 
 	// idleTimeCutoff is the amount of time we wait for an idle host to be marked as idle.
-	idleTimeCutoff            = 4 * time.Minute
+	idleTimeCutoff = 4 * time.Minute
+	// outdatedIdleTimeCutoff is the amount of time we wait for an outdated idle host to be marked idle.
+	outdatedIdleTimeCutoff    = 2 * time.Minute
```

Comment:
Why not make this even lower?

Reply:
The host could run older tasks so I do think it's okay to wait a bit for an older task. That being said I halved it arbitrarily; I'd be fine making it a minute or something.

Reply:
Ah. One minute feels better to me because the idea of keeping a host around that's not currently running something that can only run a subset of all our tasks feels less than ideal. But I'm easily swayed, either way is fine with me.